extensions.json

```diff
@@ -797,10 +797,7 @@
   "ms-toolsai.jupyter": {
     "repository": "https://github.com/microsoft/vscode-jupyter",
     "custom": [
-      "npm ci",
-      "python3 -m pip --disable-pip-version-check install -t ./pythonFiles/lib/python --no-cache-dir --implementation py --no-deps --upgrade -r requirements.txt --no-user",
-      "npm run clean",
-      "npm run package"
+      "source ~/.nvm/nvm.sh && nvm use && nvm install-latest-npm && npm install --global @vscode/vsce && npm ci && python3 -m pip --disable-pip-version-check install -t ./pythonFiles/lib/python --no-cache-dir --implementation py --no-deps --upgrade -r requirements.txt --no-user && npm run clean && npm run package"
```

Comment:
1. Can we separate the commands into lines, like it was before?
2. We can remove the nvm commands for sourcing and `nvm use`? Those should be automatically executed [[1](https://github.com/curbengh/openvsx-publish-extensions/blob/ae65295b43ba00a11c487e82c3f9a143615ef160/publish-extension.js#L68-L74)] - if that isn't working, please let me know, we should fix there. 
3. `vsce` will always be installed in the context of the publishing GitHub Action [[1]](https://github.dev/open-vsx/publish-extensions/blob/e5a086ac7ba52dfbcd80be3153cc27a9d5d44d12/.github/workflows/publish-extensions.yml#L45)

Reply:
Thanks @filiptronicek. The issue does appear to be that although `nvm install` is run, it isn't used in the following exec commands for the custom block. You can see this in [this test run](https://github.com/jared-nussbaum-frbb/publish-extensions/actions/runs/4287026987/jobs/7467368863) that it still uses the system node version. The reasoning behind the format I had chosen is because the `nvm use` would only last for the one python exec shell session so I thought this would be better than sourcing and running `nvm use` each time.

Reply:
To summarize
1. We can have each major command like `npm ci`, `npm run package` on its own line. However we would need to resolve point 2 in order to avoid having to run `source ~/.nvm/nvm.sh && nvm use` at the start of those lines
2. We currently cannot get remove the nvm commands for sourcing and `nvm use`. While `nvm install` is run, there is never a switch over to the installed node version by nvm. There is also the issue where we can't just run `nvm use` once in the `custom` section because each command [[1](https://github.com/curbengh/openvsx-publish-extensions/blob/master/publish-extension.js#L72-L79)] is run in a separate bash session [[2](https://github.com/curbengh/openvsx-publish-extensions/blob/master/lib/exec.js#L24-L33)] which resets the node that runs to the system node version. Fixing this issue should remove the need for this PR.
3. vsce install can be taken out of this PR

Additionally to avoid the need to run `nvm install-latest-npm` it might make sense to run `nvm install` with the `--latest-npm` flag

Dealing with all these points will bring us back to the existing `custom` block so feel free to close/decline this PR in favor of another PR to address those points.